proposals/stage-2/SUPPORTER_PROGRAM/SPEC.md

```diff
@@ -0,0 +1,58 @@
+# JavaScriptLandia - Phase 1 - Spec
+## October 2020 
+
+### Summary
+JavaScriptLandia is a place where all JavaScript developers, whether if you're all about TypeScript, a React evangelist, or you stick to Node.js, gather to declare their support for the JavaScript ecosystem and OpenJS. 
+
+### Phase 1 Components
+
+**Launch Announcement
+
+This will be a blog post on the OpenJS Site
+Developed in partnership with the content team at the LF 
+We will determine the best date to post with the content team
+
+Design requrements:
+* Image of a country with multiple JS Libraries
+* Image will be created by LF internal team, we also can source iterations of it on Twitter or Github
+
+**Website Changes
+
+A signup page where supporters can become members and pay for their yearly membership. We plan to adopt the auto join functionality from the Linux Foundation. 
+
+An automated workflow to add new supporters to monthly email
+* Existing subscribers will not be removed
+* They can request removal at any time
+* Zapier or IFTT can work if no other solution
+
+A new platform for the weekly emails, and a designed template. 
+    
+A page to reflect current supporters on the website
+
+A swag store for supporters (turnkey)
+
+**A Logo and Badge
+
+Inspirations and Suggestions:
+* An anthropromorphic passport, with JS branding
+* A little playful
+* Binocs
+* Fanny pack
+* Tourguide flags
+* Map
+* Passport person is traveling to see everything, doesn’t have a destination
+* Topography instead of countries
+* Play on the passport is the digital badge
+
+
+Logo Things to avoid:
```

Comment:
Great to have this section, here. Thank you!

Reply:
Agreed. Thanks.